src/core/core.cpp

```diff
@@ -0,0 +1,62 @@
+#include <vector>
+#include <nlohmann/json.hpp>
+#include <string>
+
+using json = nlohmann::json;
+
+
+struct Product {
+    std::string name;
+    int amount;
+    double price;
+    int whoBoughtID;
+};
+
+struct User {
+    int id;
+    std::string name;
+    std::string phone;
+};
+
+void parseJsonData(const json& jsonData, std::vector<Product>& products, std::vector<User>& users) {
+    for (const auto& receiptData : jsonData["receipt"]) {
```

Comment:
what if ["receipt"] field will not exist in json ?
Code will crash or everything will be fine ?

Reply:
I haven't received what json will look like, so I followed the example from trello, but yes without  ["receipt"] it won't work

Reply:
i change it and now its won't crush when something is missing